src/main/java/pl/Lukasz/Calories/Controller/MealsController.java

```diff
@@ -0,0 +1,66 @@
+package pl.Lukasz.Calories.Controller;
+
+import org.springframework.beans.factory.annotation.*;
+import org.springframework.data.repository.query.Param;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import pl.Lukasz.Calories.Entity.Meals;
+import pl.Lukasz.Calories.Entity.Users;
+import pl.Lukasz.Calories.Service.MealsService;
+import pl.Lukasz.Calories.Service.UsersService;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+@RestController
+public class MealsController {
+    @Autowired
+    private MealsService service;
+
+    @GetMapping("/meals")
+    public List<Meals> list(@Param("user") Integer user) {
+        return service.listAll(user);
+    }
+
+    @GetMapping("/meals/calories")
+    public Integer sumCaloriesToday(){
+        return service.sumCaloriesToday();
+    }
+
+    @GetMapping("/meals/{id}")
+    public ResponseEntity<Meals> get(@PathVariable Integer id) {
+        try {
+            Meals meals = service.get(id);
+            return new ResponseEntity<Meals>(meals, HttpStatus.OK);
+        } catch (NoSuchElementException e) {
+            return new ResponseEntity<Meals>(HttpStatus.NOT_FOUND);
+        }
+    }
+
+    @GetMapping("/meals/find")
+    public List<Meals> findMeals(@Param("dateFrom") String dateFrom, @Param("dateTo") String dateTo, @Param("timeFrom") String timeFrom, @Param("timeTo") String timeTo, @Param("user") Integer user ){
+        return service.findMeals(dateFrom, dateTo, timeFrom, timeTo, user);
+    }
+
+    @PostMapping("/meals/{id}")
+    public ResponseEntity<?> update(@RequestBody Meals meals, @PathVariable Integer id) {
```

Comment:
Couple of things here:
- You first download existing meal and then you don't do anything with it - so maybe it's not needed?
- PUT is a better method for updates
- It should be shallow update - e.g. if user provides just JSON with name - it should just update meals name, not remove other properties
- if you do not return anything - you should return 204
- better error handling